src/sentry/static/sentry/app/views/inviteMember/inviteMember.jsx

```diff
@@ -39,15 +39,31 @@ const InviteMember = React.createClass({
     let {slug} = this.getOrganization();
     this.api.request(`/organizations/${slug}/members/me/`, {
       method: 'GET',
-      success: ({roles}) => {
-        this.setState({roleList: roles, loading: false});
-        if (!roles || roles.filter(({allowed}) => allowed).length === 0) {
+      success: resp => {
+        let {roles} = resp || {};
+
+        if (roles.filter(({allowed}) => allowed).length === 0) {
```

Comment:
if roles is undefined, would this throw and not get to the capture message?

Reply:
Ah yeah sorry